frontend/src/common/components/LinkOutsideRouter/index.js

```diff
@@ -0,0 +1,46 @@
+import CSSModules from 'react-css-modules';
+import PropTypes from 'prop-types';
+import React from 'react';
+import browserHistory from '../../utils/routerUtils';
+
+import styles from './styles.scss';
+
+const propTypes = {
+    to: PropTypes.string.isRequired,
+    className: PropTypes.string,
+
+    children: PropTypes.oneOfType([
+        PropTypes.arrayOf(PropTypes.node),
+        PropTypes.node,
+    ]).isRequired,
+};
+
+const defaultProps = {
+    iconName: '',
+    className: '',
+};
+
+@CSSModules(styles, { allowMultiple: true })
+export default class LinkOutsideRouter extends React.PureComponent {
+    static propTypes = propTypes;
+    static defaultProps = defaultProps;
+
+    onClick = (e) => {
+        e.preventDefault();
+        browserHistory.push(this.props.to);
+
+        document.body.click();
+    }
+
+    render() {
+        return (
+            <a
+                href={this.props.to}
+                onClick={this.onClick}
+                className={this.props.className}
```

Comment:
Instead of using className, styles can be overridden from parent using "styles" prop

Reply:
We tried that, but the underlying styles would be overridden too, so we went with this.

Reply:
So this is the new way to override styles? I don't like this CSSModules.

Reply:
I think both can be done, depending on the case